### DISCUSSION TITLE
Responsive Background Colour Control for the Container Widget

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#23078] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like to be able to use different background colors for different screen sizes.

Currently, the container background image control is responsive. The background image color control is not responsive.

### Describe the solution you'd like

A responsive background color control for the container widget.

**Comment by 79ho3ein** [comment]
+1
In general, it is important to be able to control colors in all widgets in a responsive manner.

**Comment by lavendaniel** [comment]
+1 for this. I have a container that needs to be a different colour on mobile to make its contents legible. This is basic functionality that is missing.
